.release-it.json

```diff
@@ -0,0 +1,10 @@
+{
+  "src": {
+    "tagName": "v%s"
+  },
+  "buildCommand": "npm run build",
+  "github": {
+    "release": true,
+    "draft": true
+  }
```

Comment:
Add also `"requireCleanWorkingDir": false` like [here](https://github.com/uploadcare/uploadcare-redactor/blob/master/.release-it.json#L15) to allow uncommitted files in git stage